transcendence-app/src/user/user.controller.ts

```diff
@@ -242,7 +242,7 @@ export class UserController {
   unfollowUser(
     @GetUser() user: User,
     @Param('userId', ParseUUIDPipe) followedUserId: string,
-  ): Promise<Friend> {
+  ): Promise<void> {
```

Comment:
Why do you remove the return object of these two methods? I see that across the rest of our codebase, we do return the object when we delete, or add, and it makes sense, because we enable the app that consumes the API (in this case webapp), to check that the API is doing what it expects to do.

Reply:
In this case, we could check the response status -> 206 (No Content) when the request is successful and 422 (Unprocessable Entity) or 404 (Not Found) when there was an error.

Similar to [Github API](https://docs.github.com/en/rest/users/followers?apiVersion=2022-11-28#follow-a-user)